packages/modules/cart/src/migrations/Migration20250411131500.ts

```diff
@@ -0,0 +1,15 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20250326151602 extends Migration {
```

Comment:
todo: could you run `migration:create` instead of the agent writing the migration 😃 here it misses the snapshot update